datajoint/autopopulate.py

```diff
@@ -322,6 +338,7 @@ def _populate1(
                     )
                     if jobs is not None:
                         jobs.complete(self.target.table_name, self._job_key(key))
+                    return True
```

Comment:
Not sure about this `True`. The docstring needs an update to explain. It's a bit odd to have so many different return types.

Reply:
What about returning `True/False` for success/failure instead of None?

Reply:
When does it ever return `False`?

Reply:
Then you should add `return False` to the case when the function returns without populating anything.

Reply:
Does not populating anything constitute a `False`? `False` seems to indicate failure (maybe that's just me thinking so).

Reply:
The current function produces `True` if successful, `None` if nothing is populated, throws an exception, or returns `(key, error)`. It never returns `False`. It seems that if a function returns `True`, it should also return `False` in other cases. Perhaps instead of `None`, we should return `False`. It would not indicate an error, but simply not populating for other reasons than an error.

Reply:
Agreed, I'll update accordingly